components/cards/gifts/components/add-to-wishlist-form.tsx

```diff
@@ -14,25 +13,30 @@ type AddToWishListFormProps = {
 
 function AddToWishListForm({ giftId, wishlistId }: AddToWishListFormProps) {
   const { toast } = useToast();
-  const router = useRouter();
 
   const handleAddGiftToWishList = async (formData: FormData) => {
     if (!wishlistId) {
-      router.push('/register');
+      window.location.href = '/register';
       return;
     }
 
     const addToWishListWithId = addGiftToWishList.bind(null, wishlistId);
     const response = await addToWishListWithId(formData);
 
+    console.log(response);
+
     toast({
       title: response.status,
       description: response.message,
       action: (
-        <FaCheck
-          color={response.status === 'Error' ? 'red' : 'green'}
-          fontSize="36px"
-        />
+        <Button
+          onClick={() => window.location.href = '/dashboard'}
```

Comment:
Trata de usar router.push si es que es posible

Reply:
ok no problem, just using window.location.href beacuse i often find it a lot faster than router.push